apps/etl-func/src/adapters/__tests__/message.test.ts

```diff
@@ -31,10 +36,25 @@ const messageContentMock: MessageContentProvider = {
 
 const messageAdapter = new MessageAdapter(messageContentMock, loggerMock);
 
-const tokenizeMock = vi.fn();
-const tokenizerClientMock = {
-  tokenize: tokenizeMock,
-} as unknown as PDVTokenizerClient;
+const recipientStore = new Map<string, unknown>();
+
+const tokenizerClient: Mocked<TokenizerClient> = {
+  tokenize: vi
+    .fn()
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
is this comment needed?

Reply:
if i remove it the linter will fail because the `_fiscalCode` variable is not used